reactor-core/src/main/java/reactor/util/Metrics.java

```diff
@@ -125,7 +125,7 @@ public synchronized ScheduledExecutorService apply(Scheduler scheduler, Schedule
 						return schedulerId + "-" + seqNum;
 					});
 
-			/* FIXME return */ ExecutorServiceMetrics.monitor(globalRegistry, service, executorName,
+			ExecutorServiceMetrics.monitor(globalRegistry, service, executorName,
```

Comment:
I'd maybe mention the Micrometer issue in a `//TODO` so that we don't forget about it later?

Reply:
👍